docs/en/stack/ml/df-analytics/ml-dfa-classification.asciidoc

```diff
@@ -13,6 +13,13 @@ classifying malicious and benign domains to detect DGA activities for security
 reasons or predicting customer churn based on customer calling data. 
 {classification-cap} is for predicting discrete, categorical values.
 
+
+image::images/classification-vis.png[alt="Classification process",align="center"]
+The different shapes represent different classes which are devided by the line 
```

Comment:
In my opinion, we might be able to shorten this, along these lines:

> Classification is a machine learning process that predicts the class or category of a data point in a data set.
> For a simple example, consider how you might learn to differentiate circles from triangles in this graph:
> /image/
> In reality, classification problems are more complex, such as classifying malicious….

Reply:
@lcawl Updated in 0c59410